src/Widgets/IconChooser.vala

```diff
@@ -0,0 +1,89 @@
+const double M_PI = 3.14159265358979323846;
```

Comment:
This was an immediate red flag when [`Math.PI`](https://valadoc.org/glib-2.0/GLib.Math.PI.html) exists, but also, I doubt you'll need to be doing this math in the first place.

Reply:
> I doubt you'll need to be doing this math in the first place.

Why not? Using a pi constant is standard for invocations of `cairo_arc` as shown in the documentation: https://www.cairographics.org/manual/cairo-Paths.html#cairo-arc

Though presumably it would be better to use the existing `Math.PI`.

Reply:
@ids1024 I doubt the need to do this math when avatar widgets and GTK style classes exist to make circles. :wink:

Reply:
The circular GTK class doesn't work, but Hdy.Avatar might.